### ISSUE TITLE
Github Project and Pull Request not Saved in Draft

### ISSUE THREAD

**Issue by eastmaels** [utopian-io/utopian.io#66]
I'm not sure if this is the intended behavior or not, but one thing I've observed when I was writing my development contribution was the Github project and Pull Request fields were not saved in my drafts.

I know that the site will be revamped, so I'm not sure if this will still be retained. I was planning of creating an analysis contribution, but since steemsql is currently out of sync with the blockchain. I'd like to create a bug hunting contribution for now.

#### 1. Expected behavior

After saving a draft with the Github project and Pull Request fields inputted, move to another page, and selecting the draft from my drafts, the Github project and Pull Request fields should be populated as I saved them.

![image.png](https://res.cloudinary.com/hpiynhbhq/image/upload/v1510203125/a0zg3t80aqgyspngmvmf.png)

#### 2. Actual behavior

The **Github project** and **Pull Requests** fields were not automatically populated as I saved them

![image.png](https://res.cloudinary.com/hpiynhbhq/image/upload/v1510203226/p4iselehdpxludxhw94w.png)

##### 3.1. Pre-requisite

* You must have an open Pull Request in Github for the project you're contributing to. In my case, I have an open PR in busy.org's repository.

##### 3.2. Steps
 
1. Login to https://utopian.io
1. Click on ![image.png](https://res.cloudinary.com/hpiynhbhq/image/upload/v1510203327/pwqda5cml3g3yulrl2yn.png)
1. Select the **Development** category
1. Input the **Github project**. In this case, I inputted `busyorg/busy`.
1. Select an open **Pull Request**
1. Input the **Contribution Title**
1. Input the body
1. Input the necessary tags
1. Wait for the autosave function to complete
1. Leave the editr by going to another page
1. Go to your drafts
1. Wait for your drafts to load
1. Click on your autosaved contribution 

##### 3.3. Result

* Github project and Pull Requests field are not automatically populated based on values when it was auto-saved

![utopian_github_project_and_pull_request_not_saved.gif](https://res.cloudinary.com/hpiynhbhq/image/upload/v1510204326/chaeemh7oq1gy2wjen9p.gif)

<br /><hr/><em>Open Source Contribution posted via <a href="https://utopian.io/utopian-io/@eastmael/github-project-and-pull-request-not-saved-in-draft">Utopian.io</a></em><hr/>

**Comment by azarus** [comment]
@wehmoen This issue can be closed as pull request field is no longer available when creating a post.

**Comment by azarus** [comment]
@wehmoen I was able to reproduce this, i can confirm this is a bug.

**Comment by Deviad** [comment]
Could you assign this to me?

**Comment by wehmoen** [comment]
Yes. If you move tasks on the board make sure they are ordered by priority. Thanks

**Comment by diegopucci** [comment]
@wehmoen @Deviad removed the assignment on this task. Waiting for #384 and #383
